README.md

```diff
@@ -121,6 +121,26 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 - [vite-plugin-svelte](https://github.com/intrnl/vite-plugin-svelte) - Svelte integration.
 -->
 
+## Rollup Plugins compatible with Vite
+
+### Official already included in Vite
+
+- [@rollup/plugin-alias](https://github.com/rollup/plugins/blob/master/packages/alias) - Define and resolve aliases for bundle dependencies
+- [@rollup/plugin-commonjs](https://github.com/rollup/plugins/blob/master/packages/commonjs) - Convert CommonJS modules to ES6
+- [@rollup/plugin-dynamic-import-vars](https://github.com/rollup/plugins/blob/master/packages/dynamic-import-vars) - Resolving dynamic imports that contain variables.
+- [@rollup/plugin-json](https://github.com/rollup/plugins/blob/master/packages/json) - Convert .json files to ES6 modules
+- [@rollup/plugin-node-resolve](https://github.com/rollup/plugins/blob/master/packages/node-resolve) - Locate and bundle third-party dependencies in node_modules
+- [@rollup/plugin-typescript](https://github.com/rollup/plugins/blob/master/packages/typescript) - Integration between Rollup and Typescript
```

Comment:
Thanks for the clarification, I will remove these two entries for now.

Reply:
Thanks Evan, I included the ones that were in Vite's package.json, I should have dug deeper. 

I was thinking about [@rollup/plugin-image](https://github.com/rollup/plugins/tree/master/packages/image), from the docs looks like Vite supports importing images but it is not doing so by using this plugin. Should we include a section like: "Official plugins, with equivalent features in Vite" to list plugin-image, plugin-typescript, and plugin-node-resolve (and other similar cases)?
At least for me, it would be helpful to know that the features of these plugins are already covered (we can also link to the Vite docs for them)